app/src/main/java/org/oppia/app/home/recentlyplayed/OngoingListAdapter.kt

```diff
@@ -11,9 +16,14 @@ private const val VIEW_TYPE_SECTION_STORY_ITEM = 2
 
 /** Adapter to inflate different items/views inside [RecyclerView] for Ongoing Story List. */
 class OngoingListAdapter(
+  private val activity: AppCompatActivity,
   private val itemList: MutableList<RecentlyPlayedItemViewModel>
 ) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
 
+  private val orientation = Resources.getSystem().configuration.orientation
+  private var titleIndex: Int = 0
+  private var tempPosition: Int = 0
```

Comment:
Avoid using `tempPosition` make it more contextual like `previousPosition` `currentPosition` `currentStoryItemIndex` etc.

Reply:
updated to storyGridPosition.